virtio-devices/src/balloon.rs

```diff
@@ -201,6 +203,24 @@ impl BalloonEpollHandler {
 
                 let gpa = (pfn as u64) << VIRTIO_BALLOON_PFN_SHIFT;
                 if let Ok(hva) = mem.get_host_address(GuestAddress(gpa)) {
+                    if let INFLATE_QUEUE_EVENT = ev_type {
```

Comment:
Why don't you move the entire block inside line 225? We're already checking the `ev_type`, so let's take advantage of it.